packages/insured-bridge-relayer/src/Relayer.ts

```diff
@@ -57,27 +58,27 @@ export class Relayer {
   ) {}
 
   async checkForPendingDepositsAndRelay(): Promise<void> {
-    this.logger.debug({ at: "InsuredBridgeRelayer#Relayer", message: "Checking for pending deposits and relaying" });
+    this.logger.debug({ at: "AcrossRelayer#Relayer", message: "Checking for pending deposits and relaying" });
```

Comment:
I renamed these to be more "correct" in slack.

Reply:
oh boy